templates/company/README.md

```diff
@@ -0,0 +1,41 @@
+# Aragon Company template
+
+## Usage
+
+```
+company.newInstance(id, holders, stakes)
+```
+
+- `id`: Identifier to be registered in AragonID
+- `holders`: Array of token holder addresses
+- `stakes`: Array of token stakes for holders (token has 18 decimals, multiply token amount `* 10^18`)
+
+## Deploying templates
+
+After deploying ENS, APM and AragonID, just run:
+
+```
+npm run deploy:rinkeby
+```
+
+The network details will be automatically selected by the `arapp.json`'s environments.
+
+## Permissions
+
+| App               | Permission            | Grantee       | Manager |
+|-------------------|-----------------------|---------------|---------|
+| Kernel            | APP_MANAGER           | Voting        | Voting  |
+| ACL               | CREATE_PERMISSIONS    | Voting        | Voting  |
+| Voting            | CREATE_VOTES          | Token Manager | Voting  |
+| Voting            | MODIFY_QUORUM         | Voting        | Voting  |
+| Voting            | MODIFY_SUPPORT        | Voting        | Voting  |
```

Comment:
Just to note that this is a change from the current Democracy template, where we burn this role to disallow people from changing the support %.

Reply:
Yes, this was definitely intentional. 

I don't see a good reason why support % should have ever been discarded.